code/game/gamemodes/steal_items.dm

```diff
@@ -84,6 +86,7 @@
 	protected_jobs = list("Chief Engineer")
 	altitems = list(/obj/item/photo)
 	location_override = "the Chief Engineer's Office"
+	extra_information = "Stealing a photograph of the item is also an option."
```

Comment:
```suggestion
	extra_information = "Obtaining a photograph of the blueprints is also an option."
```
Minor wording change + explicitly saying it's the blueprints you want to photograph.